### DISCUSSION TITLE
SKY130 SRAM cache does not support family:1RW

### DISCUSSION THREAD

**Discussion by m-usama-z** [ucb-bar/hammer#723] [Support (Q&A)] [answered]
Hi. When I try to run the following command with a recent version of sky130 and hammer scripts,
`make buildfile tech_name=sky130 CONFIG=TinyRocketConfig`
I get the following error:
`[sram_generator] SKY130 SRAM cache does not support family:1RW`
This error does not appear when I do this with a few months old scripts in chipyard v1.7.1. It's true that Skywater SRAM cache does not have 1rw family, but it does have 1rw1r SRAMs; aren't these a sufficient substitute? Or could it be me who has misconfigured something? Kindly advise.

**Comment by nayiri-k** [comment, marked as answer]
Hello! This has happened to me before, when I accidentally ran the make command without specifying sky130 as the technology, so it defaulted to asap7 and start generating the 1RW SRAMs for that technology. Try completely removing the generated-src/ and build/ directories and starting over.

**Comment by m-usama-z** [comment]
Ok. Thanks.
